clifford/_multivector.py

```diff
@@ -496,10 +496,10 @@ def __str__(self) -> str:
             else:
                 if coeff < 0:
                     sep = seps[1]
-                    abs_coeff = -round(coeff, p)
+                    abs_coeff = -np.round(coeff, p)
                 else:
                     sep = seps[0]
-                    abs_coeff = round(coeff, p)
+                    abs_coeff = np.round(coeff, p)
```

Comment:
This seems risky - it causes a crash for `str(2**64 * e1)`.

Reply:
So I did this to fix the deprecation warning:
```
clifford/test/test_complex.py: 53 warnings
  C:\Users\hughad01\internship\clifford\clifford\_multivector.py:499: DeprecationWarning: The Python built-in `round` is deprecated for complex scalars, and will raise a `TypeError` in a future release. Use `np.round` or `scalar.round` instead.
    abs_coeff = -round(coeff, p)
```
Should we use the scalar.round option instead?

Reply:
That doesn't exist for the case I give above either

Reply:
Can we either back out this change, or add a `if np.issubdtype(self.value.dtype, np.number` around the use of `np.round`?